src/mainwindow.cpp

```diff
@@ -1665,8 +1665,10 @@ void MainWindow::initStyling() {
                          "{background-color: %1;}")
                          .arg(noteTagFrameColorName);
 
-    qApp->setStyleSheet(appStyleSheet);
-    Utils::Gui::updateInterfaceFontSize();
+//     qApp->setStyleSheet(appStyleSheet);
```

Comment:
Why can or should this be disabled?

Reply:
This happens twice, "setStyleSheet", once here once inside the updateInterfaceFontSize() function, So maybe we can just do it once? But maybe we will now need to pass in the stylesheet to the function..

Reply:
updateInterfaceFontSize uses the stylesheets that were set by the line you disabled and updateInterfaceFontSize will be executed later too... if you want to remove that line you need to make sure the styles will be set in updateInterfaceFontSize and none of the styles from appStyleSheet are supposed to be updated later (because then those styles would get overwritten by updateInterfaceFontSize again and again).

"setStyleSheet" happens in some other places too.

Reply:
If you want to play safe, just leave the line. 😁

Reply:
But why does it need to be reset again? Why not just once?

Reply:
> But why does it need to be reset again?

Who "it"?

Reply:
The stylesheet of course, and the subsequent calls to updateInterfaceFontSize. Why not just once? The function sounds to me like something we would only do once per start.

Reply:
on_overrideInterfaceFontSizeGroupBox_toggled and on_interfaceFontSizeSpinBox_valueChanged in the settings dialog

Reply:
I've added the command back in until there is a better solution. 😉